doc/antora/modules/reference/nav.adoc

```diff
@@ -38,7 +36,7 @@
 
 *** xref:unlang/local.adoc[Local Variables]
 
-*** xref:unlang/module.adoc[Modules]
+*** xref:unlang/module.adoc[Module]
```

Comment:
I think all of the plurals should stay.  Whil ethe filename is "module", the documentation describes all of the modules.